### PR TITLE
Fix string pool lookup performance

### PR DESCRIPTION
**Pull request by josephbirkner**
The naive lookup using a lower-case string copy is now really starting to impact performance during model building quite badly. So, here is a fix :smile:

### CODE REVIEW

File: src/model/string-pool.cpp [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
+
+template<typename CharT, typename Traits>
+struct TextTraits<std::basic_string_view<CharT, Traits>>
+{
+    using TValue = typename ContainerTraits<
+        std::basic_string_view<CharT, Traits>>::TValue;
+    // string is automatically null-terminated
+    static constexpr bool addNUL = false;
+
+    // is is not 100% accurate, but for performance reasons assume that string
```
Comment (johannes-wolf) [inline src/model/string-pool.cpp:33]: What is the difference?

**Review by johannes-wolf — COMMENTED** [review]
LGTM.

File: src/model/string-pool.cpp (hunk 1 above)
Comment (josephbirkner) [reply src/model/string-pool.cpp:33]: Hmm, I copied this comment from the bitsery code. I will remove it, I agree that it is confusing.

**Review by johannes-wolf — APPROVED** [review]
:1st_place_medal:

**Pull request merged**